actionview/lib/action_view/helpers/cache_helper.rb

```diff
@@ -134,7 +134,7 @@ module CacheHelper
       #
       #   <%= render @notifications, cache: false %>
       def cache(name = {}, options = nil, &block)
-        if controller.perform_caching
+        if controller.try(:perform_caching)
```

Comment:
Yeah, but if for some case receiver is not there (and it should always be) we will not catch the problem.